dashboard/test/models/school_test.rb

```diff
@@ -134,6 +134,19 @@ class SchoolTest < ActiveSupport::TestCase
     assert school.afe_high_needs?
   end
 
+  test 'AFE high needs true when school is rural' do
+    school = create :school
+    school.school_stats_by_year << SchoolStatsByYear.new(
+      {
+        school_id: school.id,
+        school_year: '1998-1999',
+        community_type: 'rural_distant'
```

Comment:
this would be `rural = 1`, because there are different community types that qualify as rural (e.g. rural_fringe, rural_remote, town_distant, town_remote)

Reply:
Oh since the logic of determining rural is based on the field `community_type` I wanted to have a better end-to-end-like test by using one of the arbitrary values. I happened to pick that one but I can also update the test to check each one in a loop if that seems better?

Reply:
Ah, I see, all good then.